src/main/java/org/toradocu/translator/BasicTranslator.java

```diff
@@ -7,23 +7,19 @@
 import org.toradocu.extractor.DocumentedExecutable;
 import org.toradocu.extractor.ParamTag;
 import org.toradocu.extractor.ThrowsTag;
-import org.toradocu.translator.spec.ExcPostcondition;
-import org.toradocu.translator.spec.Guard;
-import org.toradocu.translator.spec.Precondition;
 
 /**
  * The {@code BasicTranslator} class holds the {@code translate()} methods for {@code BlockTag} of
  * kind Param and Throws, since the translation process needed for this two tags is the same.
  */
 public class BasicTranslator {
```

Comment:
A better explanation is needed so that reader knows what is `basic` about this translator.

Why does it return a String?